crates/ironrdp-pdu/src/rdp/client_info.rs

```diff
@@ -609,12 +609,13 @@ impl Default for PerformanceFlags {
     }
 }
 
-#[repr(u16)]
+#[repr(transparent)]
 #[derive(Debug, Copy, Clone, PartialEq, Eq, FromPrimitive, ToPrimitive)]
```

Comment:
issue?: Just noticed about that: we can probably remove these traits.

```suggestion
#[derive(Debug, Copy, Clone, PartialEq, Eq)]
```

Instead provide functions such as `from_u16` and `as_u16`

Reply:
I removed traits and added implementation infallible `from_u16` and `to_u16`.